_rules/important-letter-spacing-wide-enough-24afc2.md

```diff
@@ -97,71 +97,55 @@ This `p` element has a [computed][] [letter-spacing][] of `3px`, which is **wide
 </p>
 ```
 
-#### Passed Example 4
+#### Passed Example 3
 
-This `p` element has two [declared][] values for its `letter-spacing` property. The latest wins the [cascade sort][]. It has a value of `0.15em`, and is **wide enough**.
+This text node has two [declared][] values for its `letter-spacing` property. The latest wins the [cascade sort][]. It has a value of `0.15em`, which is wide enough.
 
 ```html
 <p style="letter-spacing: 0.1em !important; letter-spacing: 0.15em !important">
 	The toy brought back fond memories of being lost in the rain forest.
 </p>
 ```
 
-#### Passed Example 5
+#### Passed Example 4
 
-This `p` element has two [declared][] values for its `letter-spacing` property. The one which is [important][] wins the [cascade sort][]. It has a value of `0.15em`, and is **wide enough**.
+This text node has two [declared][] values for its `letter-spacing` property. The one which is [important][] wins the [cascade sort][]. It has a value of `0.15em`, which is wide enough.
 
 ```html
 <p style="letter-spacing: 0.15em !important; letter-spacing: 0.1em">
 	The toy brought back fond memories of being lost in the rain forest.
 </p>
 ```
 
-#### Passed Example 6
-
-The [cascaded][] value of the `letter-spacing` property of this `p` element is [declared][] in the style sheet, not in the `style` attribute (it wins the [cascade sort][] because it is [important][]). Thus, the `p` element matches the **cascade** condition.
-
-```html
-<style>
-	p {
-		letter-spacing: 0.1em !important;
-	}
-</style>
-
-<p style="letter-spacing: 0.15em">
-	The toy brought back fond memories of being lost in the rain forest.
-</p>
-```
-
-#### Passed Example 7
+#### Passed Example 5
 
-The [computed][] value of the `letter-spacing` property of this `p` element is **not [important][]**. The [computed][] value of the `letter-spacing` property of this `span` element is the [inherited][] value, that is the [computed][] value of its parent and therefore also **not [important][]**.
+This text node has a [computed][] `letter-spacing` of `2px`, 0.2 times its [computed][] `font-size` of `10px`
 
 ```html
-<p style="letter-spacing: 0.1em">
-	<span style="letter-spacing: inherit !important;">
+<div style="font-size: 16px; letter-spacing: 2px !important">
```

Comment:
Don't think font-size is needed on this one.

Reply:
🤔 I prefer keeping it rather than relying on default `font-size`, especially given that the example is about changing the `font-size` between the definition of `letter-spacing` and the text.
Otherwise, we might need a somewhat clumsy description to point out the rewrite (e.g. saying that `2px` is not big enough for the default `16px` but that is OK because the actual one is `10px`).